### ISSUE TITLE
Expose merged config in request

### ISSUE THREAD

**Issue by innovate-invent** [vuex-orm/plugin-axios#98]
The change from version 0.5 removed the ability to inject url parameters into the request for the model.

It is still possible to recover some of this by setting in the model:
```javascript
static apiConfig = {
    url: '/foo/',
};
```
and only use api().request({}).
I would like to recover all of the functionality by exposing the built config in the Request object returned by api(). This will let me modify the `url` property to inject parameters before calling request({}).

The alternative is to restore the original functionality by accepting an object in place of the url for the Request functions that will check if `url` has been set in the config and replace its keys in the url path with its values. 

Update:
I just noticed I can monkeypatch the Request object with my own functions and that the config isn't available until the method call:
https://github.com/vuex-orm/plugin-axios/blob/5c6f51135a58c4caa8b40234f8dd303c75626da2/src/api/Request.ts#L117-L123

**Comment by kiaking** [comment]
Are you trying to set base url? Not so sure what is your proposed behavior here. Could you give me an example of the API?

**Comment by innovate-invent** [comment]
I am trying to set the axios `url` config. This is different from the baseUrl.

I ended up completely re-implementing the Request functions in my model:
https://github.com/brinkmanlab/galaxy-client/blob/f9d493674cdb7e11987b0baebbd0b56860702303/src/api/_common.js#L59-L105

The goal is to allow the model to build the url in an inheritance chain and recover the url building functionality of the previous version of this plugin.

**Comment by cuebit** [comment]
@innovate-invent is there anything further you'd like to add to this issue?

**Comment by innovate-invent** [comment]
@cuebit what more would you like me to add?

**Comment by cuebit** [comment]
@innovate-invent let me rephrase, is this issue now resolved?

**Comment by JKrol** [comment]
I have similar question. Currently, each time I want to call API, I have to provide the URL. It would be great to have an option, to define the URL on the model level.

Example - lets say I have a User model. Now I want to call `get` on the user - I have to write:
```
User.api().get('/api/users');
```
If I have many places in code where I want to get users, I have to provide '/api/users' URL in each place. It would be great to have a possibility to define URL in User model and then just call:
```
User.api().get();
```

Is this possible?

**Comment by cuebit** [comment]
@JKrol sure, you can define common options at model level [as documented](https://vuex-orm.github.io/plugin-axios/guide/configurations.html#model-configuration).

However, simply invoking `User.api().get()` without a URL is not possible. It's not how axios functions.

If you want pre-configured or shorthand methods to simply invoke anywhere in your application, consider using [Custom Actions](https://vuex-orm.github.io/plugin-axios/guide/custom-actions.html#when-to-use-custom-actions) which are designed to accommodate your exact requirements.
